shell/ptty_windows.go

```diff
@@ -11,6 +13,25 @@ import (
 	"github.com/gruntwork-io/terragrunt/options"
 )
 
+func PrepareConsole() {
```

Comment:
Please add a comment explaining what this method does

Reply:
Added